tests/group_functions/CMakeLists.txt

```diff
@@ -1,3 +1,37 @@
+function(configure_test_case)
+    cmake_parse_arguments(CTS
+	    "" "TYPE;IN_FILENAME;OUT_FILENAME;TEST_LIST" "" ${ARGN})
```

Comment:
Could you fix your editor config to avoid mixing tabs and spaces?

Reply:
@keryell I will. How many spaces are expected for CMake files? I find some are using 4 spaces but others are using 2 spaces

Reply:
Actually even this file is using both in the same file. :-(
See line 22 & 34.
2 spaces seem enough but I let Our Owner @bader decide about it.

Reply:
Let's go with two spaces. 
We have some cmake files with 4 spaces, but I don't think we use tabs.
We can unify the style across all cmake files in a separate PR.

Reply:
@keryell @bader Done. Thank you for pointing out